pandas/plotting/_core.py

```diff
@@ -1306,8 +1306,8 @@ def hist(self, by=None, bins=10, **kwargs):
 
         Examples
         --------
-        When we draw a dice 6000 times, we expect to get each value around 1000
-        times. But when we draw two dices and sum the result, the distribution
+        When we draw a die 6000 times, we expect to get each value around 1000
+        times. But when we draw two dice and sum the result, the distribution
```

Comment:
```suggestion
        When we roll a die 6000 times, we expect to get each value around 1000
        times. But when we roll two dice and sum the result, the distribution
```

Reply:
Yes, good suggestion! For completeness we might want to specify that the die is six-sided... but this feels like overkill.